shared/src/version/mod.rs

```diff
@@ -50,6 +51,10 @@ impl NetworkVersion {
     pub const V17: Self = Self(17);
     /// Hygge (builtin-actors v10)
     pub const V18: Self = Self(18);
+    /// Lightning (builtin-actors v11)
+    pub const V19: Self = Self(19);
+    /// Lightning (builtin-actors v11)
+    pub const V20: Self = Self(20);
```

Comment:
I don't understand why we need to introduce 20 right now, afaik the fip 61 PR just references nv19, but it seems ok to do so.

Reply:
Reason is tests